nasa-gibs/worldview
# wv-2579 build process eslint override removal

## Description
Our .eslintrc files (worldview\.eslintrc & worldview\web\.eslintrc) have a lot of overrides. These rules should be turned back on and the errors/warnings addressed.

This PR addresses most of the eslint overrides for the build process (worldview\.eslintrc).

Fixes # wv-2579

## How To Test
- git checkout wv-2579-eslint-overrides
- npm ci
- npm run build
- Confirm the build process succeeds
- Confirm that WV loads & runs correctly (E2E tests pass, so a quick look around should be sufficient)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
